exercises/robot-simulator/src/test/java/RobotTest.java

```diff
@@ -0,0 +1,275 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public final class RobotTest {
+
+    @Test
+    public void testRobotIsCreatedWithCorrectInitialPosition() {
+        final GridPosition initialGridPosition = new GridPosition(0, 0);
+        final Robot robot = new Robot(initialGridPosition, Orientation.NORTH);
+
+        final GridPosition actualGridPosition = robot.getGridPosition();
+
+        assertTrue(actualGridPosition.equals(initialGridPosition));
```

Comment:
I think `assertEquals` delegates to the `Object`-based `equals` implementation, so this test would fail if I used it. I can add an explicit failure message here as an alternative?